src/project.cc

```diff
@@ -341,6 +341,8 @@ std::vector<Project::Entry> LoadFromDirectoryListing(Config* init_opts,
     e.directory = config->project_dir;
     e.file = file;
     e.args = GetCompilerArgumentForFile(file);
+    if (e.args.empty())
```

Comment:
Move this to GetCompilationEntryFromCompileCommandEntry?

Reply:
It seems that other callers of `GetCompilationEntryFromCompileCommandEntry` do not have this issue.